test/integration/cases_test.go

```diff
@@ -35,6 +37,89 @@ import (
 	jobset "sigs.k8s.io/jobset/api/jobset/v1alpha2"
 )
 
+var _ = Describe("Nodepool metrics", func() {
+	Context("When reconciling a resource", func() {
+		ctx := context.Background()
+
+		jsRef := types.NamespacedName{
+			Name:      "test-jobset",
+			Namespace: "default",
+		}
+
+		jobRef := types.NamespacedName{
+			Name:      "test-job",
+			Namespace: "default",
+		}
+
+		nps, err := gkeClient.ListNodePools(ctx)
+		if err != nil {
+			Fail("Failed to list node pools: " + err.Error())
+		}
+		var np = nps[0]
+
+		var node = &corev1.Node{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "test-node",
+				Labels: map[string]string{
+					"cloud.google.com/gke-nodepool": "test-nodepool",
```

Comment:
Maybe define a constant for "test-nodepool" and use that wherever it is used.

Reply:
also moved tpu top to a const